### DISCUSSION TITLE
MONAI Label: segmentation of organ at risk in head and neck CT images

### DISCUSSION THREAD

**Discussion by GhTara** [Project-MONAI/MONAI#3310] [Q&A]
Dear MONAI,

My goal is to segment organs at risk from **CT images** of the brain such as left/right eye, left/right lens, left/right optic nerve, optic chiasm, brain stem,... overall about **17** structures. recently I have got to know about **MONAI Label** and I tried it with a **3D slicer** but I got an error! I don't know exactly why but, I think because of my dataset (they are in Dicom format).
so I have two questions: first, I don't know I am on the right way so, is the way I am trying, will have an effective outcome?
second, for segmenting these 17 structures, does exist a public model or I should train my own model? 
I am new to this field and MONAI, please guide me.

Thank you

**Comment by Nic-Ma** [comment]
Hi @SachidanandAlle ,

I think this is a question about MONAILabel, could you please help move to the `MONAILabel` repo and share some comments?

Thanks in advance.

**Comment by finalelement** [comment]
Hi @GhTara 

Thanks for raising the question for MONAI Label. Please start it off here at this [link](https://github.com/Project-MONAI/MONAILabel/discussions)

For us to be able to help you, please provide additional more information when starting off the new discussion on MONAI Label:

- What error did you get? 
- How many 3D Volumes do you have?
- What is the way that you are trying?
- Given that they are structures of the brain, I do not expect any off the shelf model to work directly. 
- Deepgrow 2D might be worth a shot. If not then starting off training of these new structures using the pretrained weights that we already have would be quite beneficial for the model that you train.

**Reply by GhTara** [reply]
Hi @finalelement 

Thank you for your guidance.

great, I will do it.
